Diferencia_Medias.py

```diff
@@ -52,7 +52,10 @@ def menu_diferencia_medias():
     n1 = int(input("n1: "))
     n2 = int(input("n2: "))
     alpha = float(input(Salpha + ": "))
-    alpha = 0.5 - alpha
+    if opcion=="3":
+        alpha = 0.5 - alpha/2
+    else:
+        alpha = 0.5 - alpha
     z = (x_bar1 - x_bar2 - delta) / (math.sqrt((sigma1/n1) + (sigma2/n2)))
```

Comment:
```suggestion
    z = (x_bar1 - x_bar2 - delta) / (math.sqrt((sigma1**2 / n1) + (sigma2**2 / n2)))
```
los sigmas van al cuadrado por fórmula